CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Table of Contents
 -  [2.0.0](#200---tbd)
+ - [2.0.0-beta.2](#200-beta2---202108015)
```

Comment:
This is only a table of contents - each entry here needs to correspond to a section later in the document.

There is no `2.0.0-beta.2` section in the document yet.

Reply:
broken link:
```suggestion
 - [2.0.0-beta.2](#200-beta2---20210815)
```

also be mindful of the comment below - the release date for beta.2 will probably be the 16th, not the 15th